core/linker/default.h

```diff
@@ -100,21 +100,35 @@ ENTRY(main_entry)
 #if !defined(STACK_SIZE_NP)
 #if defined (__ARM_FEATURE_CMSE) && (__ARM_FEATURE_CMSE == 3U)
 #define STACK_SIZE_NP 256 /* TODO: choose better default. */
+#define TOTAL_STACK_SIZE_NP STACK_SIZE_NP + STACK_GUARD_BAND
```

Comment:
Put the define inside () :  (STACK_SIZE_NP + STACK_GUARD_BAND)

Reply:
done